test/extended/fixtures/forcepull-false-s2i-is.json

```diff
@@ -0,0 +1,14 @@
+{
+    
+    "kind": "ImageStream",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "s2i-forcepull-false-ruby-20-centos7",
+        "creationTimestamp": null
+    },
+    "spec": {},
+    "status": {
+        "dockerImageRepository": ""
+    }
+    
+}
```

Comment:
I was having trouble combining objs in a single json file ... @bparees gave me a hint on how to figure out the syntax of combining ... i'll try that and see about combining.